staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -48,8 +49,24 @@ import (
 
 const execInfoEnv = "KUBERNETES_EXEC_INFO"
 
-var scheme = runtime.NewScheme()
-var codecs = serializer.NewCodecFactory(scheme)
+var (
+	scheme = runtime.NewScheme()
+	codecs = serializer.NewCodecFactory(scheme)
+
+	execCallCount = prometheus.NewCounter(
```

Comment:
Aren't metrics in a client library odd?

Reply:
There is no good way i can think of to measure this outside of client-go.
It's buried under layers of abstraction (`k8s.io/client-go/rest.Config` -> `k8s.io/client-go/transport.Transport` -> `k8s.io/client-go/transport.TLSConfig`).
And since exec plugin caches, we can't get exec count and real latency.

Do you have any ideas/suggestions about other approaches?